app/components/post_textbox/post_textbox.js

```diff
@@ -116,8 +113,8 @@ export default class PostTextbox extends PureComponent {
     }
 
     blur = () => {
-        if (this.refs.input) {
-            this.refs.input.blur();
+        if (this.input?.current) {
+            this.input.current.blur();
```

Comment:
I don't think the `?.` in `this.input?.current` is necessary here since `this.input` is defined in the constructor.

If we do need it though, this could be shortened further by removing the if statement and just using `this.input?.current?.blur();`

Reply:
you cannot call a function using `current?.blur()` as eslint complaints so I'll change it to `if (this.input.current)` instead.